src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -125,6 +138,7 @@ export const AddEmployeePage = () => {
               <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
                 <AddAffiliationPanel />
                 <UserRolesSelector
+                  personHasNin={!values.user.nvi?.verifiedAt.id}
```

Comment:
Er det ikke enklere å droppe å rendre komponenten i det heletatt når vi ikke bryr oss om roller?
```suggestion
                {!values.user.nvi?.verifiedAt.id && <UserRolesSelector
```

Reply:
ah, kanskje for å få inn fallback-teksten om det ikke er fødselsnummer det..

Reply:
Yeah, tenkte det var greit å vise headeren også.